ibis/backends/pandas/tests/test_client.py

```diff
@@ -5,14 +5,12 @@
 from pytest import param
 
 import ibis
-
-from .. import Backend
-from ..client import PandasTable  # noqa: E402
+from ibis.backends.pandas.client import PandasTable  # noqa: E402
 
 
 @pytest.fixture
 def client():
-    return Backend().connect(
+    return ibis.pandas.connect(
```

Comment:
Why this change?

Reply:
For the same reason as the other `import pandas` addition: configuration options aren't registered unless `ibis.pandas` is imported, and when tests are run in a non-deterministic order that occasionally means `ibis.pandas` isn't executed before this fixture is used to execute things.